cobalt/dom/media_source_attachment.h

```diff
@@ -32,20 +39,40 @@ class MediaSourceAttachment
  public:
   typedef web::UrlRegistry<MediaSourceAttachment> Registry;
 
-  explicit MediaSourceAttachment(scoped_refptr<MediaSource> media_source)
-      : media_source_(media_source) {}
+  MediaSourceAttachment() = default;
+  ~MediaSourceAttachment() = default;
```

Comment:
nit: turn this into "~MediaSourceAttachment() override = default;" if this works.  This explicitly signals that the dtor is virtual (It inherits its virtualness from script::Traceable).